closed/OpenJ9.gmk

```diff
@@ -329,8 +329,10 @@ else # OPENJ9_ENABLE_CUDA
 endif # OPENJ9_ENABLE_CUDA
 
 ifeq (true,$(OPENJ9_ENABLE_INLINE_TYPES))
+  CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_STRICT_FIELDS=ON
   CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_VALUE_TYPES=ON
 else # OPENJ9_ENABLE_INLINE_TYPES
+  CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_STRICT_FIELDS=OFF
   CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_VALUE_TYPES=OFF
 endif # OPENJ9_ENABLE_INLINE_TYPES
```

Comment:
Should this not also enable/disable `J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES`?